functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectConfigurator.java

```diff
@@ -119,6 +135,18 @@ private void createConfigurationFile() {
         dumpConfiguration(configFilePath);
     }
 
+    private void createConfigurationFile(String customConfigFile) {
+        Path configFilePath = Paths.get(root.toString(), customConfigFile);
+        if (!Files.exists(configFilePath)) {
+            try {
+                Files.createFile(configFilePath);
+            } catch (IOException e) {
+                throw new UncheckedIOException("Failed creating custom configuration file.", e);
+            }
+        }
```

Comment:
I'm asking myself - what is the difference between this custom way using the `createConfigurationFile(String customConfigFile)` method and the default way using the `createConfigurationFile()` method -  there is no such a check of the presence of the custom file